assets/js/modules/tagmanager/datastore/accounts.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * modules/tagmanager data store: accounts.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import Data from 'googlesitekit-data';
+import { STORE_NAME, ACCOUNT_CREATE } from './constants';
+import { isValidAccountSelection } from '../util/validation';
+import { createFetchStore } from '../../../googlesitekit/data/create-fetch-store';
+const { createRegistrySelector } = Data;
+
+// Actions
+const RESET_ACCOUNTS = 'RESET_ACCOUNTS';
+
+const fetchGetAccountsStore = createFetchStore( {
+	baseName: 'getAccounts',
+	controlCallback: () => API.get( 'modules', 'tagmanager', 'accounts', null, { useCache: false } ),
+	reducerCallback: ( state, accounts ) => {
+		return {
+			...state,
+			accounts: [ ...accounts ],
+		};
+	},
+} );
+
+export const BASE_INITIAL_STATE = {
+	accounts: undefined,
+};
+
+export const baseActions = {
+	*resetAccounts() {
+		const { dispatch } = yield Data.commonActions.getRegistry();
+
+		yield {
+			payload: {},
+			type: RESET_ACCOUNTS,
+		};
+
+		dispatch( STORE_NAME ).invalidateResolutionForStoreSelector( 'getAccounts' );
+	},
+
+	*selectAccount( accountID ) {
+		invariant( isValidAccountSelection( accountID ), 'A valid accountID is required to select.' );
+
+		const registry = yield Data.commonActions.getRegistry();
+		registry.dispatch( STORE_NAME ).setAccountID( accountID );
+
+		if ( ACCOUNT_CREATE === accountID ) {
+
```

Comment:
My editor is stripping the return there because it's an eslint violation (unnecessary return). Surprisingly it doesn't complain about the empty conditional 😆 

Will add when adding the cascading selections there.